holopy/inference/tests/test_minimizer.py

```diff
@@ -74,12 +88,31 @@ def test_iter_limit():
     holo = calc_holo(schema, cluster, 1.33, .66, illum_polarization=(1,0))
 
     #trying to do a fast fit:
-    guess1 = Sphere(center = (prior.Uniform(5, 25, guess = 15), prior.Uniform(5, 25, 15), prior.Uniform(5, 25, 20)), r = (prior.Uniform(.4, .6, guess = .45)), n = 1.59)
-    guess2 = Sphere(center = (prior.Uniform(5, 25, guess = 14), prior.Uniform(5, 25, 14), prior.Uniform(5, 25, 20)), r = (prior.Uniform(.4, .6, guess = .45)), n = 1.59)
-    par_s = Spheres([guess1,guess2])
+    guess1 = Sphere(
+        center=(prior.Uniform(5, 25, guess=15),
+                prior.Uniform(5, 25, 15),
+                prior.Uniform(5, 25, 20)),
+        r=(prior.Uniform(.4, .6, guess=.45)),
+        n=1.59)
+    guess2 = Sphere(
+        center=(prior.Uniform(5, 25, guess=14),
+                prior.Uniform(5, 25, 14),
+                prior.Uniform(5, 25, 20)),
+        r=(prior.Uniform(.4, .6, guess = .45)),
+        n=1.59)
+    par_s = Spheres([guess1, guess2])
 
-    model = AlphaModel(par_s, medium_index=1.33, illum_wavelen=.66, illum_polarization=(1, 0), alpha = prior.Uniform(.1, 1, .6))
+    model = AlphaModel(
+        par_s, medium_index=1.33, illum_wavelen=.66, illum_polarization=(1, 0),
+        alpha=prior.Uniform(.1, 1, .6))
     warnings.simplefilter("always")
```

Comment:
do you know why this is here? It seems an odd spot to be messing around with warnings. When running tests, HoloPy is configured to have warnings raise exceptions. It's probably not a good idea to have individual tests alter this behaviour unless using `with warnings.catch_warnings*()` as you do immediately below.

Can you delete it without making tests fail?

Reply:
Should be fixed by af57daf.

However there are 2 other spots where a ```simplefilter``` is set w/o a catch statement:

1. In ```holopy.core.tests.common```, it says:
```python
# tests should fail if they give warnings
import warnings
warnings.simplefilter("error")
```
2. In ```holopy.scattering.scatterer.spherecluster``` (i.e. not in a test), it says:
```python
# default to always warning the user about overlaps.  This can be overriden by
# calling this function again with a different action.
warnings.simplefilter('always', OverlapWarning)
```

I think the second is correct, but I'm not sure about the first. Is that what we want? How do we know that ```core.tests.common``` is imported first?

Reply:
You raise a good point. I think the intention was that all test files import something from `holopy.core.tests.common`, thereby running that line of code. There are currently 9 test files in HoloPy that do not follow that pattern, most of which we recently created/refactored:

```
holopy/core/tests/test_errors.py
holopy/inference/tests/test_cma.py
holopy/inference/tests/test_prior.py
holopy/scattering/tests/test_composite.py
holopy/scattering/tests/test_lowlevel.py
holopy/scattering/tests/test_mielens.py
holopy/scattering/tests/test_mielensfunctions.py
holopy/scattering/tests/test_scatterer.py
holopy/scattering/tests/test_tmatrix.py
```
I'm not sure what the best solution is...

Reply:
Will be solved by #302